lib/internal/Magento/Framework/ObjectManager/Test/Unit/Code/Generator/_files/Sample.php

```diff
@@ -16,6 +16,11 @@ class Sample
      */
     protected $messages = [];
 
+    /**
+     * @var array
+     */
+    protected $config = [];
```

Comment:
Please make it private, like any new property.

Reply:
Changed to private

Reply:
Anyway, it is a test sample file, it is not used anywhere and changing this has no impact on unit tests.

Reply:
Thanks, I understand, still conventions should be obeyd ;)